contracts/BatchExchangeViewer.sol

```diff
@@ -1,6 +1,8 @@
 pragma solidity ^0.5.0;
+pragma experimental ABIEncoderV2;
```

Comment:
I don’t know if this is supported by current versions of `web3` and `ethabi` and hence `ethcontract`. I’ll have to check.

Reply:
Let me know. If it's supported I'd love to refactor the orderbook fetching to also return list of structs instead of byte arrays (only in the Viewer contract though as we don't want experimental features in the core contract).

Reply:
I will go ahead and merge this now. As we are not depending on it yet it shouldn't cause any incompatibilities and we can change it later on if we need a different encoding for ethcontracts

Reply:
Any news on the support check @nlordell ?

Reply:
Sorry, forgot to update here. No this is currently not supported by either `web3` crate or `ethcontract` :cry:

Reply:
Does this mean even the other methods cannot be deserialized or only the ones that would be returning a struct? I don't know if ABIEncoderV2 changes the encoding for "traditional" return types as well.